interfaces/builtin/dbus.go

```diff
@@ -89,6 +89,13 @@ dbus (bind)
     bus=###DBUS_BUS###
     name=###DBUS_NAME###{_,-}[1-9]{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9]}{,_[1-9]{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9_]}{,[0-9]}},
 
+# For Firefox, support using org.mozilla.firefox.<id> as the 'well-known' name
+# where <id> is the base64-encoded profile name.
+# See https://bugzilla.mozilla.org/1441894 for a discussion and details.
+dbus (bind)
+    bus=###DBUS_BUS###
+    name="###DBUS_NAME###.*",
```

Comment:
On the one hand this looks ok, and we have a bit similar open rule accommodating KDE applications. On the other hand the KDE rule has some structure (i.e. we expect somethign that looks like a sequence of numbers in the suffix), while this one only has a known prefix, which then can be followed by anything. Maybe that's ok given there's some implicit namespace concept attached to dbus names.